deps/minizip/zip.c

```diff
@@ -1862,7 +1851,7 @@ int Write_EndOfCentralDirectoryRecord(zip64_internal* zi, uLong size_centraldir,
    return err;
 }
 
-int Write_GlobalComment(zip64_internal* zi, const char* global_comment)
+static int Write_GlobalComment(zip64_internal* zi, const char* global_comment)
 {
   int err = ZIP_OK;
   uInt size_global_comment = 0;
```

Comment:
I wasn't sure whether to actually touch minizip or not since it's a dependency, but I figure it might as well be cleaned up since it's compiled into our binary. Maybe I should run astyle on it too.